common/src/test/java/com/google/auto/common/SuperficialValidationTest.java

```diff
@@ -274,7 +274,7 @@ public void invalidAnnotationValue() {
   }
   */
 
-  private abstract static class AssertingProcessor extends AbstractProcessor {
```

Comment:
Um... dunno - but it got flipped internally.  Maybe we need to un-flip it.  :/  I'll revert this one change, and we can push it the other direction then.